gluster_exporter/conf/conf.go

```diff
@@ -0,0 +1,34 @@
+package conf
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+var Collectors = make(map[string]CollectorConfig)
+
+type CollectorConfig struct {
+	Name         string `json:"name"`
+	SyncInterval uint64 `json:"sync_interval"`
+	Enabled      bool `json:"enabled"`
```

Comment:
Can this be flipped as below? so that this entry required in config file only if a report needs to be disabled

    Disabled  bool `json:"disabled"`

Reply:
Sure. Will do that change.